Ivan_Shytskyi/Lesson_16/Lesson_16.Homework/CreateVote.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson_16.Homework
+{
+    internal class CreateVote : Candidate
```

Comment:
Your CreateVote class should not inherit Candidate->Person. This is another entity in your business domain

Reply:
Name CreateVote fits bad here because it is name of an action but not business entity. Should be named as 'VoteManager' or simple 'Vote'